libs/designsystem/calendar/src/helpers/calendar-cell.model.ts

```diff
@@ -1,9 +1,18 @@
-export interface CalendarCell {
+export interface CalendarCell extends CalendarCellMetadata {
   date: number;
   monthIndex: number;
   year: number;
+  ariaLabel: string;
+}
+
+export interface CalendarCellMetadata {
```

Comment:
Consider renaming - it feels like this interface represents metadata for a day more than a cell:
```suggestion
export interface CalendarDayMetadata {
```